src/core/toga/widgets/optioncontainer.py

```diff
@@ -137,10 +141,26 @@ def content(self):
         """
         return self._content
 
+    @property
+    def number_of_tabs(self):
+        return len(self.content)
```

Comment:
I don't think this entry point is needed; `len()` is the Pythonic way of establishing size of a container; since the user has access to the container, I don't think we need an explicit API to return the size of that container.